yeonjae22/FBMember/app/src/main/java/user/manage/fbmember/ModifyActivity.kt

```diff
@@ -0,0 +1,29 @@
+package user.manage.fbmember
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import com.google.firebase.auth.FirebaseUser
+import kotlinx.android.synthetic.main.modify.*
+
+class ModifyActivity : AppCompatActivity(){
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.modify)
+
+        val accountManager = AccountManager.getInstance()
+
+        updateBtn.setOnClickListener {
+            accountManager.updateUser(modifyPassword.text.toString(), modifyName.text.toString(), object : AccountManager.CallBack{
```

Comment:
```suggestion
            accountManager.updateUser("${modifyPassword.text}",  "${modifyName.text}", object : AccountManager.CallBack{
```
이렇게 사용해보세요. 굉장히 유용한 코틀린 syntax입니다.

Reply:
변경했습니다!